src/main/java/seedu/linkedout/model/applicant/JobContainsKeywordsPredicate.java

```diff
@@ -1,14 +1,16 @@
 package seedu.linkedout.model.applicant;
 import java.util.List;
+import java.util.function.Predicate;
 
 import seedu.linkedout.commons.util.StringUtil;
 
 /**
  * Tests that a {@code Applicant}'s {@code Name} matches any of the keywords given.
  */
-public class JobContainsKeywordsPredicate extends KeywordsPredicate {
+public class JobContainsKeywordsPredicate extends KeywordsPredicate implements Predicate<Applicant> {
 
     private final List<String> keywords;
+    private int noOfMatches;
```

Comment:
Just to clarify, where is this used. Is it supposed to be the numberMatches in test?

Reply:
oh sorry I forgot to delete this while testing my code, thanks for noticing it